### PR TITLE
examples: Update NI-Digital, NI-DMM, NI-Fgen, NI-Switch, NI-Scope, NI-VISA and Output voltage readme to describe the .env simulation process

### PR DESCRIPTION
**Pull request by Avinash2Suresh**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Updated the `NI-Digital`, `NI-DMM`, `NI-Fgen`, `NI_Switch`, `NI-Scope`, `NI-VISA` and `Output voltage` example's readme files to describe the simulation process.

### Why should this Pull Request be merged?

Explains the process of enabling the simulation in the readme files.

### CODE REVIEW

**Review by dixonjoel — APPROVED** [review]

**Review by keer1111 — APPROVED** [review]

**Review by vigkre — APPROVED** [review]

File: examples/nivisa_dmm_measurement/README.md [hunk 1]
```diff
@@ -39,9 +39,15 @@ Supported instrument models:
 - NI Instrument Simulator v2.0
 - HP/Agilent/Keysight 34401A DMM
 
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
+- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically registered measurement services).
+- Add the following options to the `.env` file to enable simulation via the driver's option string:
```
Comment (bkeryan) [inline examples/nivisa_dmm_measurement/README.md:44]: The `visa_dmm` instrument driver doesn't have an option string. Also, there is only one option.

```suggestion
- Add the following option to the `.env` file to enable simulation via the driver's `simulate` parameter:
```

File: examples/output_voltage_measurement/README.md [hunk 2]
```diff
@@ -39,11 +39,20 @@ Supported instrument models:
 - NI Instrument Simulator v2.0
 - HP/Agilent/Keysight 34401A DMM
 
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
+- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically registered measurement services).
+- Add the following options to the `.env` file to enable simulation via the driver's option string:
```
Comment (bkeryan) [inline examples/output_voltage_measurement/README.md:44]:
```suggestion
- Add the following options to the `.env` file to enable simulation via the driver's option string or `simulate` parameter:
```

File: examples/niswitch_control_relays/README.md [hunk 3]
```diff
@@ -26,5 +26,11 @@ This is a MeasurementLink example that controls relays using an NI relay driver
 
 This example requires an NI relay driver (e.g. PXI-2567).
 
-By default, this example uses a simulated instrument. To use a physical instrument, edit
-`_constants.py` to specify `USE_SIMULATION = False`.
\ No newline at end of file
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
+- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically registered measurement services).
+- Add the following options to the `.env` file to enable simulation via the driver's option string:
```
Comment (bkeryan) [inline examples/niswitch_control_relays/README.md:31]: NI-Switch supports an option string, but not via the `niSwitch_InitializeWithTopology` entry point that we are using. It uses separate parameters.

```suggestion
- Add the following options to the `.env` file to enable simulation via the driver's `simulate` and `topology` parameters:
```

File: examples/nidigital_spi/README.md [hunk 4]
```diff
@@ -25,5 +25,12 @@ This is a MeasurementLink example that tests an SPI device using an NI Digital P
 
 This example requires an NI Digital Pattern instrument (e.g. PXIe-6570).
 
-By default, this example uses a simulated instrument. To use a physical instrument, edit
-`_constants.py` to specify `USE_SIMULATION = False`.
\ No newline at end of file
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
```
Comment (bkeryan) [inline examples/nidigital_spi/README.md:28]: Sorry to rewrite this again, but "create a simulated device" implies that it adds the device to NI MAX, but it doesn't. Also, we're mixing "instrument" and "device" here.

```suggestion
By default, this example uses a physical instrument or a simulated instrument created in NI MAX. To automatically simulate an instrument without using NI MAX, follow the steps below:
```

File: examples/nidmm_measurement/README.md [hunk 5]
```diff
@@ -24,5 +24,12 @@ This is a MeasurementLink example that performs a measurement using an NI DMM.
 
 This example requires an NI DMM (e.g. PXIe-4081).
 
-By default, this example uses a simulated instrument. To use a physical instrument, edit
-`_constants.py` to specify `USE_SIMULATION = False`.
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
+- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically registered measurement services).
+- Add the following options to the `.env` file to enable simulation via the driver's option string:
```
Comment (bkeryan) [inline examples/nidmm_measurement/README.md:29]: Did you notice that the rest of the content in these files is word wrapped? Please word-wrap the new content so it looks nice in a text editor. I have been using the Rewrap extension for VS Code: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: examples/nidmm_measurement/README.md (hunk 5 above)
Comment (dixonjoel) [reply examples/nidmm_measurement/README.md:29]: Which column have you been using for wrapping in .md files?

Comment (dixonjoel) [reply examples/nidmm_measurement/README.md]: I wrapped with column 100.

File: examples/nidigital_spi/README.md (hunk 4 above)
Comment (bkeryan) [reply examples/nidigital_spi/README.md]: > This applies to all of these files, including nidcpower_source_dc_voltage.

I'll fix this

File: examples/nidmm_measurement/README.md (hunk 5 above)
Comment (bkeryan) [reply examples/nidmm_measurement/README.md]: I've been using the default.

File: examples/nivisa_dmm_measurement/README.md [hunk 6]
```diff
@@ -26,22 +28,35 @@ and a DMM that supports SCPI commands.
 
 > **Note:**
 >
-> This example uses the custom instrument driver `_visa_dmm.py` to perform the device-specific commands and queries.
+> This example uses the custom instrument driver `_visa_dmm.py` to perform the
+> device-specific commands and queries.
 
 ### Required Hardware
 
 By default, this example does not require hardware; it uses PyVISA-sim to
```
Comment (bkeryan) [inline examples/nivisa_dmm_measurement/README.md:36]: This is not true anymore. I'll fix it.

**Review by bkeryan — APPROVED** [review]

File: examples/nidmm_measurement/README.md (hunk 5 above)
Comment (bkeryan) [reply examples/nidmm_measurement/README.md]: I rewrapped the files using the default setting, which uses about 80 columns.

File: examples/nidcpower_source_dc_voltage/README.md [hunk 7]
```diff
@@ -29,18 +31,24 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
...
   ```
   MEASUREMENTLINK_NIDCPOWER_SIMULATE=1 
   MEASUREMENTLINK_NIDCPOWER_BOARD_TYPE=PXIe
   MEASUREMENTLINK_NIDCPOWER_MODEL=4141
   ```
 
-> **Note**
-> The multi-site pin map, `NIDCPowerSourceDCVoltageMultiSite.pinmap`, requires an
-> NI SMU with 4 or more channels.
+> **Note** The multi-site pin map, `NIDCPowerSourceDCVoltageMultiSite.pinmap`,
```
Comment (bkeryan) [inline examples/nidcpower_source_dc_voltage/README.md:53]: *sigh*, Rewrap messes up the GitHub markdown "note" formatting

**Pull request merged**